<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Setting {
background-image: url(:/BG/setting/setting page.jpg);
}

QPushButton#position {
background-color: transparent;
border-image: url(:/Button/setting/position.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#time {
background-color: transparent;
border-image: url(:/Button/setting/time.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#run {
background-color: transparent;
border-image: url(:/Button/setting/run.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide1{
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide1:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide2 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide2:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide3 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide3:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide4 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide4:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide5 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide5:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide6 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide6:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide7 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide7:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide8 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide8:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#slide9 {
background-color:transparent;
border-image: url(:/Button/main menu/off button.png);
background: none;
border: none;
background-repeat: none;
}

QPushButton#slide9:checked {
border-image: url(:/Button/main menu/on button.png);
}

QPushButton#back {
background-color:transparent;
border-image: url(:/Button/setting/back.png);
background: none;
border: none;
background-repeat: none;
}

QLabel#x{	
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}
QLabel#y{
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}
QLabel#z{
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}
QLabel#heater{
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}
QLabel#cylinder{
	font: 18pt &quot;MS Shell Dlg 2&quot;;
}

</string>
  </property>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="position">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="time">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>310</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_heater">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_rotate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_suction">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>718</y>
     <width>680</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="holder_x">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>227</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="holder_y">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>227</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="holder_z">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>227</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="heater">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>227</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cylinder">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>227</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="slide9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
